src/encoding/text.rs

```diff
@@ -33,15 +33,98 @@ use std::borrow::Cow;
 use std::collections::HashMap;
 use std::fmt::Write;
 
+/// Functions to encode metrics to the OpenMetrics text format in parts.
+/// Useful when assembling metrics from multiple registries.
+/// Prefer calling [`encode`](encode) when encoding a single registry.
+///
+/// Here is a simple example:
+/// ```
+/// # use prometheus_client::encoding::text::parts;
+/// # use prometheus_client::metrics::counter::Counter;
+/// # use prometheus_client::metrics::histogram::{Histogram, exponential_buckets};
+/// # use prometheus_client::registry::Registry;
+/// #
+/// # fn main() -> Result<(), std::fmt::Error> {
+/// let mut orders_registry = Registry::default();
+/// let total_orders: Counter<u64> = Default::default();
+/// let processing_times = Histogram::new(exponential_buckets(1.0, 2.0, 10));
+/// orders_registry.register(
+///     "total_orders",
+///     "Total orders received",
+///     total_orders.clone(),
+/// );
+/// orders_registry.register(
+///     "processing_times",
+///     "Millisecond order times",
```

Comment:
Please always use SI units, i.e. here seconds. See OpenMetrics specification. Histograms are using `f64`, thus no precision is lost.

Reply:
Done.